### DISCUSSION TITLE
y axis horizontal scroll

### DISCUSSION THREAD

**Discussion by arif3210** [Abhinandan-Kushwaha/react-native-gifted-charts#826] [Q&A] [answered]
How to make y axis and y axis label stay static (not getting scrolled) while scrolling horizontally in a line chart?

**Comment by Abhinandan-Kushwaha** [comment]
Hi @arif3210 Y-axis and Y-axis labels do NOT scroll. This is the default behaviour.
Can you share a video/gif showing it scrolling? Also can you share the code for the same?

**Comment by arif3210** [comment]
https://github.com/user-attachments/assets/41ad5a71-1242-45fb-b584-297a57e033f9

What I meant is when i Scroll it horizontally, the y axis labels also scroll away, i want the labels stay and the scroll content scroll.Heres the code.Hoping this would help

      <ScrollView horizontal >
      <View style={Style.areaChartView}> 
          <LineChart 
          data={areaChartData}
          xAxisLabelTexts={areaChartLabels}
          showValuesAsDataPointsText
          color1='red'
          noOfSections={5}
          dataPointsWidth={6}
          dataPointsHeight={6}
          textShiftY={-2}
          textShiftX={-5}
          textFontSize={13}
          textColor='black'
          backgroundColor={"#B5C0D0"}
          thickness={5}
          hideRules
          focusEnabled
          />

        </View>
          </ScrollView>

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
@arif3210 You don't need the `<ScrollView>`. Remove it.

**Comment by arif3210** [comment]
Thankyou for the support. I removed scrollview, but linechart didnt scroll much, then I added width prop to linechart, it worked.
